test/smoke/html-fn.html

```diff
@@ -0,0 +1,64 @@
+ <!--
+@license
+Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<!DOCTYPE html>
+<html>
+<head>
+  <script src="../../../webcomponentsjs/webcomponents-lite.js"></script>
+  <link rel="import" href="../../polymer-element.html">
+</head>
+<body>
+  <script>
+    const html = Polymer.html;
```

Comment:
Is there a specific reason you have to do this assignment? I suspect it has to do with syntax highlighting not working on it?
```js
Polymer.html`template`
```
If this is the reason, I would add it in the documentation of the function to encourage this pattern instead. (and also point to Polymer 3 having a simple import for this, making the assignment useless)